app/tests/spec/views/sign_up.js

```diff
@@ -1341,6 +1341,15 @@ define(function (require, exports, module) {
         view.$('#submit-btn').click();
         assert.isTrue(TestHelpers.isEventLogged(metrics, 'flow.signup.submit'));
       });
+
+      it('logs the suggest-sync event', () => {
```

Comment:
Thanks for catching that, I erroneously put the element ID instead if the event name, will update.